src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(Lottery.getRandomBall());
```

Comment:
Using static methods is generally discouraged as it can lead to less flexible and less object-oriented code. Consider creating an instance of the Lottery class and calling a non-static method to get a random ball.